Geometry/HGCalCommonData/data/v1/hgcalEE.xml

```diff
@@ -110,6 +110,7 @@
   <SpecPar name="HGCalEE">
     <PartSelector path="//HGCalEESensitive.*"/>
     <Parameter name="Volume" value="HGCalEESensitive" eval="false"/>
+    <Parameter name="GeometryMode" value="HGCalGeometryMode::Square" eval="false"/>

```

Comment:
@ianna - do I understand this code correctly to say that in each xml we have a line to which "GeometryMode" we do _NOT_ use? Could we not make this more descriptive by giving the GeometryMode we DO use instead?